src/main/java/org/apache/commons/imaging/formats/pnm/PnmImageParser.java

```diff
@@ -42,13 +43,13 @@
 public class PnmImageParser extends ImageParser<PnmImagingParameters> {
 
     private static final String DEFAULT_EXTENSION = ImageFormats.PNM.getDefaultExtension();
-    private static final String[] ACCEPTED_EXTENSIONS = {
-            ImageFormats.PAM.getDefaultExtension(),
-            ImageFormats.PBM.getDefaultExtension(),
-            ImageFormats.PGM.getDefaultExtension(),
-            ImageFormats.PNM.getDefaultExtension(),
-            ImageFormats.PPM.getDefaultExtension()
-    };
+    private static final List<String> ACCEPTED_EXTENSIONS = Arrays.asList(
```

Comment:
Should this additionally wrap the list in `Collections.unmodifiableList`? Because while the list returned by `Arrays.asList` does not support changing the size, it does still support replacing elements.